<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehiclePasswordPage</class>
 <widget class="QWidget" name="VehiclePasswordPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget 
{
	background-color:black;
}
QPushButton
{
	font:30px, &quot;微软雅黑&quot;;
    color: white;
	background-color:black;
	border-radius:8px;border-top: 4px solid lightgray;	
	border-left: 4px solid lightgray;border-bottom: 1px solid lightgray;	
	border-right: 1px solid lightgray;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>610</y>
     <width>1018</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 0, 0);

	border:1px solid white;
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>620</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;

}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num1">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>190</y>
     <width>78</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>190</y>
     <width>78</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num3">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>190</y>
     <width>78</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num5">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>260</y>
     <width>78</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>260</y>
     <width>78</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num6">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>260</y>
     <width>78</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num8">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>330</y>
     <width>78</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num7">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>78</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num9">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>330</y>
     <width>78</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num0">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>400</y>
     <width>78</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NumDEL">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>400</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>DEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordLbl">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:  3px solid white;
background-color:blue;
border-radius:8px;
font:30px; 
color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:30px &quot;微软雅黑&quot;; 
color:white;</string>
   </property>
   <property name="text">
    <string>输入密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmBtn">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>620</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;

}</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="alarmlbl">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:30px &quot;微软雅黑&quot;; 
color:red;</string>
   </property>
   <property name="text">
    <string>密码错误！请重新输入！</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
